registry/polkadot.json

```diff
@@ -396,5 +396,13 @@
         "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/jeeper_delegate.png",
         "shortDescription": "Governance Researcher @opengov.watch",
         "isOrganization": false
+    },
+    {
+        "address": "1313ciB4VzPeH3n1QKJym1brBmzRdfHBEctWipgH4uGsyF6n",
+        "name": "Polkadot Poland DAO - PolskaDOT",
+        "image": "https://raw.githubusercontent.com/PolkadotPolandDAO-PolskaDOT/opengov-delegate-registry/66feb88945768f4b75bf467e2c4fb3c8e9788b37/images/Polkadot%20Poland%20DAO%20-%20PolskaDOT.png",
```

Comment:
Hello!
based our rules, all links should reach to our repository, please change it to /novasamatech/opengov-delegate-registry/

Reply:
Links corrected